package.json

```diff
@@ -84,6 +84,7 @@
     "ejs": "^2.5.6",
     "eslint-plugin-eslint-plugin": "^0.7.1",
     "eslint-plugin-node": "^4.2.1",
+    "eslint-plugin-throws-on-load": "^1.0.0",
```

Comment:
I couldn't think of a good way to test the "throwing on load" behavior (proxyquire doesn't seem to support this case), so I just published a [real plugin](https://github.com/not-an-aardvark/eslint-plugin-throws-on-load) that throws on load for the purposes of this test.